### PR TITLE
Appeals status page

### PR DESCRIPTION
**Pull request by bshyong**
Some refactoring and more changes to reach parity with mockup. Note that content is not finalized yet in screenshot below

screen shot 2017-06-21 at 6 03 31 pm

### CODE REVIEW

File: src/js/disability-benefits/utils/appeal-helpers.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+export const appealStatusDescriptions = (eventType) => {
```
Comment (U-DON) [inline src/js/disability-benefits/utils/appeal-helpers.jsx]: Why change this to a function?

**Review by U-DON — APPROVED** [review]
LGTM.

File: src/js/disability-benefits/utils/appeal-helpers.jsx (hunk 1 above)
Comment (bshyong) [inline src/js/disability-benefits/utils/appeal-helpers.jsx]: Do react nodes behave properly if this is a javascript object?  I made this a functional component which seemed to accomplish what I needed

Comment (U-DON) [inline src/js/disability-benefits/utils/appeal-helpers.jsx]: I thought they would. You're still using them inside an object in the function. I don't know if wrapping them within a function is a requirement for React nodes. But I'm not opposed to doing it either way.

Comment (bshyong) [inline src/js/disability-benefits/utils/appeal-helpers.jsx]: Object worked—in addition to importing React, needed to change the file extension to `jsx`

**Pull request merged**